cvat-ui/src/components/create-cloud-storage-page/cloud-storage-form.tsx

```diff
@@ -228,7 +226,7 @@ export default function CreateCloudStorageForm(props: Props): JSX.Element {
 
         cloudStorageData.specific_attributes = specificAttributes.toString();
 
-        if (uploadedKeyFile) {
+        if (uploadedKeyFile && uploadedKeyFile.name !== fakeCredentialsData.keyFile.name) {
```

Comment:
What if my keyfile name is ``fakeKey.json``?)

Reply:
The key file will not be updated.

Reply:
Updated